.github/workflows/gem-push.yml

```diff
@@ -0,0 +1,32 @@
+name: Ruby Gem
+
+on:
+#   push:
+#     branches: [ master ]
+  # run manually for now
+  workflow_dispatch:
```

Comment:
Eventually, could we have this linked to creating on publishing a new release?
https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release

Out of curiosity, how can we run actions manually? I've only had experience with automatically triggered ones.

Reply:
Changed to be run after a published release. 

Before, to run it manually, we'd have to go to the "Actions" tab, select the workflow, and click "Run Manually". I wanted to have this for testing, but I think it's straightforward enough to just try a release.